lib/driver.js

```diff
@@ -251,6 +255,31 @@ class AndroidDriver extends BaseDriver {
     await this.initAutoWebview();
   }
 
+  shouldDismissChromeWelcome (opts) {
+    return (!_.isUndefined(opts.chromeOptions) &&
+      !_.isUndefined(opts.chromeOptions.args) &&
+      opts.chromeOptions.args.indexOf('--no-first-run') !== -1);
+  }
+
+  async dismissChromeWelcome () {
+    log.info("Trying to dismiss Chrome welcome");
+    let activity = await this.getCurrentActivity();
+    if (activity === "org.chromium.chrome.browser.firstrun.FirstRunActivity") {
```

Comment:
I have tried chrome version from 53 to 56 and it's always the same activity name.